Userland/Libraries/LibDSP/Effects.cpp

```diff
@@ -89,4 +89,37 @@ void Mastering::process_to_fixed_array(Signal const& input_signal, FixedArray<Sa
     }
 }
 
+BandAdjustment::BandAdjustment(NonnullRefPtr<Transport> transport)
+    : EffectProcessor(transport)
+    , m_filter(move(transport))
+    , m_frequency("Frequency"_string, 20, 20'000, 300, Logarithmic::Yes)
+    , m_gain_db("Gain (dB)"_string, -60, 20, 0, Logarithmic::No)
+    , m_q("Q"_short_string, 0.02, 7, 1, Logarithmic::No)
+{
+    m_parameters.append(m_frequency);
+    m_parameters.append(m_gain_db);
+    m_parameters.append(m_q);
+
+    m_frequency.register_change_listener([this](auto const& frequency) {
+        m_filter.set_center_frequency(static_cast<double>(frequency));
+    });
+    m_gain_db.register_change_listener([this](auto const& gain_db) {
+        m_filter.set_gain_db(static_cast<double>(gain_db));
+    });
+    m_q.register_change_listener([this](auto const& q) {
+        m_filter.set_q(static_cast<double>(q));
```

Comment:
Why these casts to `double` when the argument takes in a `float`?

Reply:
Copy-paste, since we need to create a floating-point value from the fixed-point parameters.